pkg/webhook/webhook.go

```diff
@@ -40,18 +46,20 @@ import (
 // +kubebuilder:rbac:groups="apiextensions.k8s.io",resources=customresourcedefinitions,verbs=get;list
 // +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;create;update;patch;list;watch
 // +kubebuilder:rbac:groups="kubevirt.io",resources=virtualmachines,verbs=get;list;watch;create;update;patch
-var AddToManagerFuncs []func(manager.Manager, *pool_manager.PoolManager) error
+var AddToManagerFuncs []func(manager.Manager, *webhookserver.Server, *pool_manager.PoolManager) error
```

Comment:
You need to have AddToManagerFuncs and AddToWebhookFuncs

Reply:
Do you mean change the name to AddToWebhookFuncs?

Reply:
Yep, since this ones are just adding it to webhook.